v2/thirdparty/common-customizations/sign-in-and-up/custom-providers.mdx

```diff
@@ -65,6 +65,8 @@ SuperTokens.init({
 
 ## Step 2: Back End 
 
+TODO Talk about the built-in custom provider instead
+
```

Comment:
also talk about dynamically adding provider or allowing the user to add their own SSO